python/dgl/distributed/dist_context.py

```diff
@@ -228,7 +228,7 @@ def initialize(ip_config, num_servers=1, num_workers=0,
         formats = os.environ.get('DGL_GRAPH_FORMAT', 'csc').split(',')
         formats = [f.strip() for f in formats]
         rpc.reset()
-        keep_alive = os.environ.get('DGL_KEEP_ALIVE') is not None
+        keep_alive = bool(int(os.environ.get('DGL_KEEP_ALIVE', 0)))
```

Comment:
why do you need to convert it to int and then to bool?

Reply:
because `int` value is specified for `DGL_KEEP_ALIVE` and `os.environ.get()` always return string? so has to convert to int first. Then check whether the int value equals zero. it's a bit clumsy.

Reply:
How about: `keep_alive = bool(os.environ.get('DGL_KEEP_ALIVE', False))`

Reply:
if so, `DGL_KEEP_ALIVE=False` will be evaluated as `True`. namely, once macro `DGL_KEEP_ALIVE` is defined, it's always evaluated as `True`.

Reply:
Use `ast.literal_eval`?

Reply:
return type of `literal_eval` is not very  clear. I prefer to convert on our own though Current parse is a bit weird.